model/project_ref.go

```diff
@@ -1803,10 +1791,11 @@ func GetProjectSettingsById(projectId string, isRepo bool) (*ProjectSettings, er
 
 // GetProjectSettings returns the ProjectSettings of the given identifier and ProjectRef
 func GetProjectSettings(p *ProjectRef) (*ProjectSettings, error) {
-	hook, err := FindGithubHook(p.Owner, p.Repo)
-	if err != nil {
-		return nil, errors.Wrapf(err, "finding GitHub hook for project '%s'", p.Id)
-	}
+
+	// GetProjectSettings shouldn't error if GitHub app is not set up, or installed on a repo.
+	// It will return false for githubHooksEnabled though.
```

Comment:
I think we can skip this second sentence

Reply:
Should we be caching whether or not the app is installed? Can we just use the existing GithubHooks collection for this? Seems kind of expensive to keep asking github for it (even if they don't count it towards our rate limit, seems slow)

Reply:
caching installation id now